.github/actions/notary-server-ip/action.yml

```
@@ -0,0 +1,19 @@
+name: get-ips-of-service-containers
+description: 'Get alerting endpoint and notary IPs'
+outputs:
+  notary_ip:
+    description: "IP address of the notary instance"
+    value: ${{ steps.get_notary_server_ip.outputs.notary_ip }}
+runs:
+  using: "composite"
+  steps:
+    - name: Get  IP
+      id: get_notary_server_ip
+      run: |
+        NOTARY_CONTAINER=$(docker container ls --no-trunc --format "{{json . }}" | jq ' . | select(.Image|match("notary:server"))')
+        NOTARY_CONTAINER_ID=$(echo ${NOTARY_CONTAINER} | jq -r .ID)
+        NOTARY_CONTAINER_NETWORK=$(echo ${NOTARY_CONTAINER} | jq -r .Networks)
+        NOTARY_SEARCH_PATH=.[0].NetworkSettings.Networks.${NOTARY_CONTAINER_NETWORK}.IPAddress
+        NOTARY_IP=$(docker container inspect ${NOTARY_CONTAINER_ID} | jq -r ${NOTARY_SEARCH_PATH})
```

Not sure whether in theory there could be a whitespace in ${NOTARY_SEARCH_PATH}, but if there was, this would break, thus probably 
```suggestion
        NOTARY_IP=$(docker container inspect ${NOTARY_CONTAINER_ID} | jq -r "${NOTARY_SEARCH_PATH}")
```
is just safer

---

To a lesser extend (due to nature of ID), same for ${NOTARY_CONTAINER_ID}